advanced/Scripts/piholeDebug.sh

```diff
@@ -33,32 +33,15 @@ cat << EOM
 ::: and optionally upload the generated log to a unique and random directory on
 ::: Termbin.com. NOTE: All log files auto-delete after 1 month and you are the only
 ::: person who is given the unique URL. Please consider where you post this link.
-::: 
+:::
 EOM
 
-
```

Comment:
To get in line with the rest of the scripts (See #814)m `pihole` (well, and `basic-install.sh`) will be the only script to have the root check, as everything else is called by it. No need to keep checking for elevation in each script called by `pihole`! 

It also means that script's **must** be called using `pihole`, although if you wanted you could run `sudo /opt/pihole/somescript.sh`